release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -3238,4 +3238,28 @@ $ oc adm release info 4.11.26 --pullspecs
 [id="ocp-4-11-26-updating"]
 ==== Updating
 
+To update an existing {product-title} 4.11 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-11-27"]
+=== RHSA-2023:0651 - {product-title} 4.11.27 bug fix and security update
+
+Issued: 2023-02-14
+
+{product-title} release 4.11.27, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2023:0651[RHSA-2023:0651] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:0650[RHBA-2023:0650] advisory.
```

Comment:
```suggestion
{product-title} release 4.11.27, which includes security updates, is now available. The list of bug fixes that are included in the update are documented in the link:https://access.redhat.com/errata/RHSA-2023:0651[RHSA-2023:0651] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:0650[RHBA-2023:0650] advisory.
```

Reply:
Hi @aireilly . Thanks for the review. I'll retain "is" because it's associated with the word "list".